src/abstractdataframe/join.jl

```diff
@@ -247,6 +248,12 @@ Join two `DataFrame` objects
   if `true`, duplicate names will be suffixed with `_i`
   (`i` starting at 1 for the first duplicate).
 
+* `validate` : whether to check that columns passed as the on argument
+   define unique keys in each input data frame. Can be a tuple or a pair,
+   with the first element indicating whether to run check for `df1` and
+   the second element for `df2`. By default no check is performed.
+   Merge key equality it tested using `isequal`.
```

Comment:
"Merge" isn't the terminology we use. I'd just add this mention this after "in each input data frame", like this: ``(according to [`isequal`](@ref)) ``.